Gaijinizer patch/Map472.txt

```diff
@@ -1,396 +1,504 @@
 ひとつボタンを押せば、この絶望の世界から離脱できる。
 
 だけど……彼の人生はその後もずっと続くんだ……
-> CONTEXT: Map472/events/1/pages/0/201/Dialogue/enemy27 < UNTRANSLATED
-
+> CONTEXT: Map472/events/1/pages/0/201/Dialogue/enemy27
+At the push of a button, the Interveners who've
+lost themselves to the \\C[10]Old One's \\C[0]temptations
+can walk away from this world of despair. But his
+life will continue on for a long time.#Ron
 > END STRING
 
 > BEGIN STRING
 アバドーンの介入に染まってしまった彼の人生は
 独立した世界線に存在している……
 
 僕の『介入』の権限では……救ってあげることができない…
-> CONTEXT: Map472/events/1/pages/0/206/Dialogue/enemy27 < UNTRANSLATED
-
+> CONTEXT: Map472/events/1/pages/0/206/Dialogue/enemy27
+His life, now tainted with Abaddon's
+Interventions, continues in this isolated
+timeline. I can't save him with my own
+Intervention authority.#Ron
 > END STRING
 
 > BEGIN STRING
 ……ミズキちゃん
 ベッドにいるカズヤくんは、どういう表情をしているの？
 
 ここからだとよく見えなくて…
-> CONTEXT: Map472/events/1/pages/0/217/Dialogue/enemy15 < UNTRANSLATED
+> CONTEXT: Map472/events/1/pages/0/217/Dialogue/enemy15
+Mizuki-chan, what kind of expression does that
+Kazuya-kun lying in bed have?
 
+I can't really see from back here.#Ron
 > END STRING
 
 > BEGIN STRING
 …………見えない方がいいわ。
 
 きっと、彼が大好きな夜深ちゃんには
 見るに堪えない姿よ……
-> CONTEXT: Map472/events/1/pages/0/222/Dialogue/enemy38 < UNTRANSLATED
+> CONTEXT: Map472/events/1/pages/0/222/Dialogue/enemy38
+You're better off not being able to see.
 
+It'd be too much for you to handle considering
+how much you love the guy.#Ron
 > END STRING
 
 > BEGIN STRING
 私も正直、後ろで良かったわ……
 
 実はあんまり、話しについていけてなかったの……
-> CONTEXT: Map472/events/1/pages/0/227/Dialogue/enemy38 < UNTRANSLATED
+> CONTEXT: Map472/events/1/pages/0/227/Dialogue/enemy38
+I'm honestly glad that I'm in the back here.
 
+I haven't been able to keep up with what they're
+talking about.#Ron
 > END STRING
 
 > BEGIN STRING
 ……なぜ百鬼は、僕を生かしたんだ？
 
 こうなると分かっているなら、
 いっそのこと、楽にしてくれれば……
-> CONTEXT: Map472/events/1/pages/0/234/Dialogue/kazuya < UNTRANSLATED
-
+> CONTEXT: Map472/events/1/pages/0/234/Dialogue/kazuya
+Why did Hyakki let me live? If she knew it would
+end up like this, I'd have rather she put me out
+of my misery.#Ron
 > END STRING
 
 > BEGIN STRING
 ……出来なかったんだよ
 
 彼女には、君の後ろにいる\\C[10]『介入者』\\C[0]の存在を分かっていた。
 自分に立ち向かうのが、\\C[10]『介入者』\\C[0]の意思であることも……
-> CONTEXT: Map472/events/1/pages/0/239/Dialogue/enemy27 < UNTRANSLATED
-
+> CONTEXT: Map472/events/1/pages/0/239/Dialogue/enemy27
+Because she couldn't not.
+She understood both the existence of the
+\\C[10]Intervener \\C[0]behind you, as well as the will of
+theirs that she was opposing.#RonIffyWording
 > END STRING
 
 > BEGIN STRING
 そう悟った途端、殺す気が失せてしまったらしい。
 
 \\C[10]決着がつくような戦いをしてしまえば
 そのルートを辿った『介入者』の思う壺だったからね
-> CONTEXT: Map472/events/1/pages/0/244/Dialogue/enemy27 < UNTRANSLATED
-
+> CONTEXT: Map472/events/1/pages/0/244/Dialogue/enemy27
+She apparently lost any desire to kill once she
+comprehended those things. \\C[10]That's because having
+a decisive final battle would just be playing
+into the hands of the Intervener\\C[0].#Ron
 > END STRING
```

Comment:
from previous comment: (let me know if it's too much).
```
She apparently lost any desire to kill once she
comprehended those things. \\C[10]That's because
having a final battle would only just be for
the amusement of the Intervener\\C[0].
```